pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -136,6 +136,9 @@ type KogitoAppBuildObject struct {
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 	// Internal Maven Mirror to be used during source-to-image builds to considerably increase build speed
 	MavenMirrorURL string `json:"mavenMirrorURL,omitempty"`
+	//Artifact contains final generated project artifact
+	// + optional
+	Artifact Artifact `json:"artifact,omitempty"`
```

Comment:
Please add the `operator-sdk` flags to the attributes of this type.

Reply:
done

Reply:
done